src/server.ts

```diff
@@ -57,9 +58,27 @@ export function buildApp({ store, importer }: AppOptions): Application {
             if (value === null) {
               await importer.unconfigure()
             } else {
-              const election = parseElection(value)
-              await importer.configure(election)
-              await store.setElection(election)
+              const electionConfig = value as Election | ElectionDefinition
```

Comment:
This endpoint can now accept either an `Election` or `ElectionDefinition` (I don't love that name). Accepting the latter will enable us to preserve and check the hash explicitly if we want to.